recipe/recipe_test.go

```diff
@@ -0,0 +1,182 @@
+package recipe_test
+
+import (
+	"github.com/go-debos/debos/recipe"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+type testRecipe struct {
+	recipe string
+	err    string
+}
+
+// Test if incorrect file has been passed
+func TestParse_file(t *testing.T) {
```

Comment:
TestParseErrors_ ? it's a bit of an odd naming

Reply:
done